modules/oe_subscriptions_anonymous/oe_subscriptions_anonymous.module

```diff
@@ -61,7 +61,10 @@ function oe_subscriptions_anonymous_mail($key, &$message, $params) {
 function oe_subscriptions_anonymous_theme($existing, $type, $theme, $path) {
   return [
     'oe_subscriptions_anonymous_message_confirm' => [
-      'variables' => [],
+      'variables' => [
```

Comment:
The work is good, but I would:

1. Not default any values, require title/message to be passed always.
2. Change the template hook name: maybe we could name it `oe_subscriptions_anonymous_status_message` ?

Reply:
The name change is Ok. 
About the require of the title/message, to be sure, you want to make them mandatory? Like no displaying anything or thrwoing an error?

Reply:
Often Drupal uses NULL and doesn't really check anything. Let's check for each value and do not print anything if not there.

Reply:
Changed to NULL, added checks